jekyll-sitemap.gemspec

```diff
@@ -14,7 +14,7 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r!^(test|spec|features)/!)
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "jekyll", "~> 3.3"
+  spec.add_dependency "jekyll", "~> 3.4", ">= 3.4.2"
```

Comment:
The plugin doesn't break if Jekyll is less than 3.4.2, so I'd probably say it's additional functionality. Shall I remove this change then?

Shall I also modify the tests to handle both sets of functionality?